guides/assets/javascripts/responsive-tables.js

```diff
@@ -1,43 +1,43 @@
-$(document).ready(function() {
+(function() {
+  "use strict";
   var switched = false;
-  $("table").not(".syntaxhighlighter").addClass("responsive");
+
+  document.querySelectorAll(":not(.syntaxhighlighter)>table").forEach(function(element) {
```

Comment:
[`NodeList.prototype.forEach()`](https://developer.mozilla.org/en-US/docs/Web/API/NodeList/forEach) isn't supported in Internet Explorer and some older browser versions. You'll need to slice these into an array first or use a `for` loop to iterate.